src/store/modules/settings.js

```diff
@@ -1,76 +1,121 @@
+import Vue from "vue";
+
 /* eslint-disable no-alert, no-console */
 const state = () => ({
   shareLink: undefined,
   sparcApi: undefined,
-  algoliaIndex: 'k-core_dev_published_time_desc',
+  algoliaIndex: "k-core_dev_published_time_desc",
   algoliaKey: undefined,
   algoliaId: undefined,
   pennsieveApi: undefined,
   flatmapAPI: undefined,
   nlLinkPrefix: undefined,
   rootUrl: undefined,
-  facets: {'species':[], 'gender':[], 'organ':[]},
-  markers: []
+  facets: { species: [], gender: [], organ: [] },
+  markers: [],
+  featuredMarkers: [],
```

Comment:
Is there a reason not to turn featuredMarkers into a list of objects for storing the Identifiers, Dois, Species and DatasetId?

Reply:
I think I did this to make watching the list easier, rather than making deeper object changes. And then forcing Vue to be reactive.